vizro-core/tests/unit/vizro/models/_components/form/test_slider.py

```diff
@@ -17,14 +17,7 @@
 def expected_slider():
     return html.Div(
         [
-            dcc.Store(
-                "slider_id_callback_data",
-                data={
-                    "id": "slider_id",
-                    "min": 0,
-                    "max": 10,
-                },
-            ),
+            dcc.Store("slider_id_callback_data", data={"id": "slider_id", "min": 0, "max": 10}),
```

Comment:
I would also add `id` here

Reply:
I would create a separate ticket for this to go through all the tests and double-check where this doesn't apply yet. But I wouldn't do it in this PR, as this PR should only tidy the trailing commas that were added by black automatically.

Reply:
Yeah sure. If you could just mention this PR link in that ticket (for the purpose of reminding) it would be awesome.

Reply:
Already did :) https://github.com/orgs/McK-Internal/projects/15/views/12?pane=issue&itemId=51469120